docker/pytorch/Dockerfile

```diff
@@ -8,6 +8,19 @@ ARG DEBIAN_FRONTEND=noninteractive
 # won't be able to build the docker image.
 RUN rm -f /usr/local/cuda-11.3/cuda-11.3
 
+# update repository keys
+# https://developer.nvidia.com/blog/updating-the-cuda-linux-gpg-repository-key/
+ARG LINUX_DISTRO
```

Comment:
Do we need this variable, or can we compute it dynamically (say by reading `/etc/os-release`)?

Reply:
We can compute it but will need `sed` magic and combining different variable. I thought you don't like that in favor of readability.  Content of /etc/os-release is the following and we need `ubuntu1804` I used LINUX_DISTRO because it already existed in build matrix. 

```
NAME="Ubuntu"
VERSION="18.04.5 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.5 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```

Reply:
Right, my bad. I thought a new variable was being added here. Makes sense haha.